ts/sagas/startup/loadVisibleServicesHandler.ts

```diff
@@ -100,13 +102,13 @@ export function* loadVisibleServicesRequestHandler(
       // Dispatch action to remove the services from the redux store
```

Comment:
I noted a little bug in previous code (probably mine).
On line 47 this line should be removed
```typescript
yield put(
        loadVisibleServices.failure(Error(readableReport(response.value)))
      );
```
Are you agree?

Reply:
yes, removed within https://github.com/teamdigitale/io-app/pull/1352/commits/6fc4d857f3dd8046f0a0553483c77948eb4158c3